user/getting-started.md

```diff
@@ -4,38 +4,107 @@ layout: en
+The very short guide to using Travis CI with your GitHub hosted code repository. If you're new to continuous integration or would like some more information on what Travis CI does, start with [Travis CI for Complete Beginners](/user/for-beginners) instead.
 
-Or if you want a more complete guide to a particular language, pick one of these:
+<div id="toc"></div>
 
-{% include languages.html %}
+## To get started with Travis CI
 
-### To get started with Travis CI:
+You'll need a GitHub account with admin access to at least one repository, and ideally you'll already have a working build script that you run manually.
 
 1. [Sign in to Travis CI](https://travis-ci.org/auth) with your GitHub account, accepting the GitHub [access permissions confirmation](/user/github-oauth-scopes).
 
-2. Once you're signed in, and we've synchronized your repositories from GitHub, go to your [profile page](https://travis-ci.org/profile) and enable Travis CI for the repository you want to build.
+2. Once you're signed in, and we've synchronized your GitHub repositories, go to your [profile page](https://travis-ci.org/profile) and enable Travis CI for the *open source* repository you want to build. If you want to build a private repository, sign in to [Travis CI for private repositories](https://travis-ci.com/profile) instead.
 
-   > Note: You can only enable Travis CI builds for repositories you have admin access to.
+3. Add a `.travis.yml` file to your repository to tell Travis CI what to build.
 
-3. Add a `.travis.yml` file to your repository to tell Travis CI what to build:
+   This example tells Travis CI that this is a Ruby project, so unless you change the default, Travis CI uses `rake` to build it.
+
+   Travis CI tests this project against Ruby 2.2 and the latest versions of JRuby and Rubinius, which can all pass or fail independently.
 
    ```yaml
    language: ruby
    rvm:
     - 2.2
     - jruby
     - rbx-2
-   # uncomment and edit the following line if your project needs to run something other than `rake`:
-   # script: bundle exec rspec spec
    ```
 
-   This example tells Travis CI that this is a project written in Ruby and built with `rake`. Travis CI tests this project against Ruby 2.2 and the latest versions of JRuby and Rubinius.
-
 4. Add the `.travis.yml` file to git, commit and push, to trigger a Travis CI build:
 
-   > Travis only runs a build on the commits you push **after** adding the repository to Travis.
+   > Travis only runs builds on the commits you push *after* you've enabled the repository in Travis CI.
+
+5. Check the [build status](https://travis-ci.org/repositories) page to see if your build passes or fails, according to the return status of the build command.
+
+## Selecting a programming language
+
+Use one of these common languages:
+
+```yaml
+language: ruby
+language: java
+language: node_js
+language: android
+language: php
```

Comment:
👍 for adding Python.

We should drop Android from this list, given that it requires extra configuration in `android.components`.

Also, there should not be multiple `language` key-value pairs in `.travis.yml`, so I don't think we should present them like this.

Instead:

```yaml
language: ruby
```

```yaml
language: java
```

```yaml
language: node_js
```

```yaml
language: php
```

```yaml
language: python
```